.github/SUPPORT.md

```diff
@@ -0,0 +1,29 @@
+# Getting help with `{effectsize}`
+
+Thanks for using `{effectsize}`. Before filing an issue, there are a few places
+to explore and pieces to put together to make the process as smooth as possible.
+
+Start by making a minimal **repr**oducible **ex**ample using the
+[reprex](http://reprex.tidyverse.org/) package. If you haven't heard of or used
+reprex before, you're in for a treat! Seriously, reprex will make all of your
+R-question-asking endeavors easier (which is a pretty insane ROI for the five to
+ten minutes it'll take you to learn what it's all about). For additional reprex
+pointers, check out the [Get help!](https://www.tidyverse.org/help/) resource
+used by the tidyverse team.
+
+Armed with your reprex, the next step is to figure out where to ask:
+
+  * If it's a question: start with StackOverflow. There are more people there to answer questions.
+  * If it's a bug: you're in the right place, file an issue.
+  * If you're not sure: let's [discuss](https://github.com/easystats/effectsize/discussions) it and try to figure it out! If your
```

Comment:
@IndrajeetPatil I've linked to the discussion tab here

Reply:
Sounds good!